### PR TITLE
Implement cool chaining operator and cleanup mapping

### PR DESCRIPTION
**Pull request by scott062**

### CODE REVIEW

File: src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx [hunk 1]
```diff
@@ -21,16 +21,14 @@ const AgendaItemLegs = props => {
...
     <>
       {
         legs$.map((leg) => (
           <td>
             {
-              helperFunc ?
-                <dd>{helperFunc(leg[key]) ? helperFunc(leg[key]) : 'None listed'}</dd>
-                :
-                <dd>{leg[key] ? leg[key] : 'None listed'}</dd>
+              <dd>{helperFunc(leg[key]) ?? leg[key] ?? 'None listed'}</dd>
```
Comment (M00staff) [inline src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx:31]: i wasn't aware you could do as many of these as you wanted... though, this kinda works like a `||` xept its looking for a defined value, not just truthy 🤔 will be fun to play with

also neat trick is `??=`

Comment (scott062) [reply src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx:31]: nullish coalescing operators FTW 🙌

File: src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx [hunk 2]
```diff
@@ -21,16 +21,14 @@ const AgendaItemLegs = props => {
   const formatLang = (langArr = []) => langArr.map(lang => (
     `${lang.code} ${lang.spoken_proficiency}/${lang.reading_proficiency}`
   )).join(', ');
-  const getData = (key, helperFunc) => (
+
+  const getData = (key, helperFunc = () => {}) => (
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx:25]: ❤️

**Review by elizabeth-jimenez — APPROVED** [review]
👻

**Comment by elizabeth-jimenez** [comment]
image
Very nice - confirmed for read only AIM, PM AIs and AIH

**Comment by elizabeth-jimenez** [comment]
Failing cause complexity, but that's addressed in another PR, so should be g2m

**Pull request merged**